src/studio/src/designer/frontend/shared/components/molecules/AltinnInformationCardForChildren.tsx

```diff
@@ -1,83 +1,43 @@
-import { createTheme } from '@material-ui/core';
-import Grid from '@material-ui/core/Grid';
-import { withStyles } from '@material-ui/core/styles';
-import classNames from 'classnames';
 import React from 'react';
-import altinnTheme from '../../theme/altinnStudioTheme';
-import createInformationCardStyles from '../styles/createInformationCardStyles';
+import classNames from 'classnames';
+import classes from './AltinnInformationCardForChildren.module.css';
 
 export interface IAltinnInformationCardComponentProvidedProps {
   headerText: string;
   imageSource: string;
   shadow: boolean;
-  classes: any;
 }
 
-const theme = createTheme(altinnTheme);
-
-const styles = () =>
-  createInformationCardStyles(theme, {
-    breadText: {
-      paddingTop: 15,
-      fontSize: 16,
-      paddingBottom: 39,
-    },
-  });
-
-const AltinnInformationCardForChildren = (
+/**
+ *     breadText: {
+ *       paddingTop: 15,
+ *       fontSize: 16,
+ *       paddingBottom: 39,
+ *     },
+ *
```

Comment:
??

Reply:
Forgot that one from the work process, didn't know if it affected someone so I just put it there temporarily.